content/collections/source-catalog/en/snowflake.md

```diff
@@ -521,3 +521,13 @@ SELECT ...
 FROM your_table
 WHERE TO_DATE(event_timestamp) BETWEEN '2024-01-01' AND '2024-01-31'
 ```
+
+## Troubleshooting Snowflake Imports
+
+1. SQL compilation error: Invalid identifier INFORMATION_SCHEMA.QUERY_HISTORY_BY_SESSION. Results not generated.
+
+This error commonly appears in situations where a Snowflake import source used to work but suddenly stopped.
```

Comment:
nit: this error can happen anytime, so prefer to remove this since this info doesn't help debug.